StPicoEvent/StPicoBTofPidTraits.cxx

```diff
@@ -1,3 +1,13 @@
+/***************************************************************************
+ *
+ * $Id$
+ *
+ * Author: Xin Dong (xdong@lbl.gov)
```

Comment:
Thanks for removing those CVS keywords at least for now or until someone complains... Unfortunately, I anticipate that the overloading GetCVS will have to be required for StPicoDstMaker. But it is not as bad as ever expanding $Log$. GetCVS contains only the $Id$ and $Name$ tags.